### PR TITLE
🚧 (project) add rich library for a user friendly CLI

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

`ralph` can be used as a CLI and integrating the rich library makes the tool more user friendly and easy to read.

## Proposal

The [provided handler from rich](https://rich.readthedocs.io/en/stable/reference/logging.html?rich.logging.RichHandler) replaces the custom made stream handler.

:test_tube: logger tests are failing, as for now, I have not updated them with the new handler. The rich integration enforces message wrapping. It is not yet possible to run correctly the test

### CODE REVIEW

**Comment by quitterie-lcs** [comment]
Seen and discussed with @wilbrdt, we should consider configure the logger in a other way. To get the coloration with the RichHandler, a console should be configured.
Will configure the logger with a direct instantiation of the handler

File: setup.cfg [hunk 1]
```diff
@@ -54,6 +54,7 @@ cli =
     bcrypt>=4.0.0
     click>=8.1.0
     click-option-group>=0.5.0
+    rich>=13.1.0
```
Comment (wilbrdt) [inline setup.cfg:57]: 13.2.0 now released!

**Comment by quitterie-lcs** [comment]
@jmaupetit what do you think of this configuration for `rich` ?

File: .env.dist [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Configuration
 RALPH_APP_DIR=/app/.ralph
+FORCE_COLOR=1
+TERM=xterm-256
```
Comment (wilbrdt) [inline .env.dist:4]: I personally don't mind with this env var approach. Maybe add a comment just to be clear for Rich.

**Pull request closed without merging**